packages/dev-server/src/main.ts

```diff
@@ -187,6 +187,7 @@ const reloadLambdas = () => {
   console.log('Change detected, rebuilding...')
   purgeRequireCache()
   lambdaFunctions = requireLambdaFunctions(PATH)
+  console.log(`rebuild completed. Listening again on ${HOSTNAME}`)
```

Comment:
```suggestion
  console.log('Done')
```

Reply:
If you feel inspired to add a the amount of ms it took to rebuild the lambdas that would be awesome!

Reply:
Sure, no issues. It won't be a big change. Could you give me a text? 
i.e `Done in 30ms.`

Reply:
Text looks great!